ui/00_message/achievement/achievement_name.xml

```diff
@@ -6268,14 +6295,16 @@
   <message>
     <key>ACHIEVEMENT_NAME_2546</key>
     <original>目覚めた収集家</original>
-    <translation>Collector Arisen</translation>
+    <translation>Arisen Collector</translation>
```

Comment:
Well yeah, the whole "awakened/enlightened (person)" is referencing Arisen. I just don't like how `X Collector` reads unless it's very clearly an adjective, because it misreads as `Collector of X`. If you have a better way to word it, go for it.